bin/load-mocks

```diff
@@ -21,6 +21,15 @@ from pytz import utc
 
 from sentry import buffer, roles, tsdb
 from sentry.event_manager import HashDiscarded
+from sentry.incidents.logic import (
+    AlertRuleNameAlreadyUsedError,
+    create_alert_rule,
+    create_alert_rule_trigger,
+    create_incident,
+    create_incident_activity,
+    create_initial_event_stats_snapshot,
```

Comment:
These are both unused

Reply:
Dang, does the linter not lint this file?

Reply:
I guess they are warnings